src/main/java/app/web/controller/ListasController.java

```diff
@@ -28,22 +28,23 @@ public MovieList consultarLista(@RequestHeader String token, @PathVariable Long
 
     @RequestMapping(value = "/", method = RequestMethod.POST)
     @ResponseBody
-    public String crearLista(@RequestHeader String token, @RequestBody MovieList nuevaLista) throws IOException {
+    public String crearLista(@RequestHeader String token, @RequestBody String nuevaLista) throws IOException {
         /* Acá no va a devolver el string sino un void con el status 200 en caso positivo o excepción en caso de que falle la creacioN*/
-        return "Se creó la lista.";
+    	listasService.crearLista(nuevaLista);
+    	return "Se creó la lista.";
```

Comment:
Ok, voló el return String. Pero tengo una duda, en la documentación de la API dice que me devuelve esto: 
{
  "status_message": "The item/record was created successfully.",
  "id": 5854,
  "success": true,
  "status_code": 1
}

Yo no debería chequear que se creó y, a partir de esta respuesta, asignar en mi dominio el id que me viene en el json??

Con respeto al Request Body, en la documentación tengo el siguiente ejemplo:
{
  "name": "My Cool List",
  "iso_639_1": "en"
}

Entiendo que es un String con el nombre, pero el segundo String no se exactamente qué es.

https://developers.themoviedb.org/4/list/create-list